### ISSUE TITLE
The tutorial

### ISSUE THREAD

**Issue by khaledk2** [pywr/pywr#120]
I have went step by step through the tutorial (http://pywr.github.io/pywr-docs/tutorial.html) but I got this error
ImportError: cannot import name NumpyArrayRecorder
from this import statement 
from pywr._core import NumpyArrayRecorder

**Comment by snorfalorpagus** [comment]
Thanks for the bug report - I'm really keen for people like yourself to get involved with the project. The tutorial is a little behind the current build. I'll update it when I have a chance.

You need to use `pywr.recorders.NumpyArrayNodeRecorder` like this:

```
>>> from pywr.core import Model, Input, Output
>>> from pywr.recorders import NumpyArrayNodeRecorder
>>> model = Model()
>>> node = Input(model, 'supply')
>>> recorder = NumpyArrayNodeRecorder(model, node)
>>> node.recorders
[<pywr._recorders.NumpyArrayNodeRecorder object at 0x000000000634D908>]
```

Nodes can now have multiple recorders, and doing the above automatically adds the recorder to the list of recorders. To access:

```
>>> print(supply.recorder[0].data[0])
```

**Comment by snorfalorpagus** [comment]
Fixed in b31e73a8c05d09b68c1f6c47ca4d28cd1b620544.
